### PR TITLE
[NUI] Add hidden  Disposed field for BaseHandle

### PR DESCRIPTION
**Pull request by elishateng**
### Description of Change ###

[NUI] Add hidden Disposed field for BaseHandle.

### API Changes ###

 - ACR:

### CODE REVIEW

File: src/Tizen.NUI/src/public/BaseHandle.cs [hunk 1]
```diff
@@ -521,6 +521,12 @@ public class FocusRequestArgs : EventArgs
         {
             get => swigCPtr;
         }
+
+        /// <summary>
+        /// A flag to check if it is already disposed.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/BaseHandle.cs:528]: Hello, please add this~

**_/// \<remarks>Hidden API: Only for inhouse or developing usage. The behavior and interface can be changed anytime.\</remarks>_**

**Review by dongsug-song — APPROVED** [review]
looks good to me

**Review by zhouleonlei — APPROVED** [review]
LGTM

**Review by zg2nets — APPROVED** [review]
lgtm

**Review by AchoWang — COMMENTED** [review]
Looks good to me.

**Pull request merged**